bin/pv.js

```diff
@@ -36,3 +37,20 @@ function stderr (str) {
   process.stderr.cursorTo(0) // move cursor to beginning of line
   process.stderr.write(str)
 }
+
+function formatEta (eta) {
```

Comment:
should we use maybe `ms` depency to clean it a little bit and avoid having this format function?